assets/sass/components/_place-context-header.scss

```diff
@@ -1,19 +1,43 @@
 .place-context-header {
-  background-color:  govuk-colour("light-grey");
-  padding: govuk-spacing(3);
-  border-left: $govuk-border-width solid govuk-colour("blue");
-  margin-bottom: govuk-spacing(0);
+  @include govuk-responsive-padding(2, "top");
+  @include govuk-responsive-padding(2, "bottom");
+  background-color: govuk-colour("blue");  
 }
 
-.place-context-header__attribute-container {
-  > div {
-    display: inline;
-    padding-right: govuk-spacing(2);
-    margin-right: govuk-spacing(2);
-    border-right: 1px solid govuk-colour("black");
+.place-context-header__heading{
+  color: govuk-colour("white");
+}
+
+.place-context-header__attributes{
+  @include govuk-media-query($from: tablet) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+}
+
+.place-context-header__row{
+  @include govuk-media-query($from: tablet) {
+    @include govuk-responsive-margin(2, "right");
+    @include govuk-responsive-padding(2, "right");
 
-    &:last-child {
-      border-right: 0;
-    }
+    display: flex;
+    flex-wrap: wrap;
+    border-right: 1px solid govuk-colour("white");
```

Comment:
Do we want the border? Prototype does not show it, instead only using space.

Reply:
Good spot, it does feel a bit a `l` or ` | ` are added the content

![image](https://github.com/user-attachments/assets/57574bb4-df43-4f65-9c59-752385001e15)